.projenrc.js

```diff
@@ -2,12 +2,25 @@ const child = require('child_process');
 const path = require('path');
 const { JsiiProject } = require('projen');
 
+const DEFAULT_K8S_VERSION = '22';
 const SPEC_VERSION = k8sVersion();
 const K8S_VERSION = `1.${SPEC_VERSION}.0`;
 
 function k8sVersion() {
   const branch = child.execSync('git rev-parse --abbrev-ref HEAD').toString().trim();
-  return branch.replace('k8s.', '');
+  if (branch.startsWith('k8s.') && branch.length >= 6) {
```

Comment:
We need to specify what happens if the branch is not one of "k8s.22", "k8s.21", or "k8s.20" otherwise 'npx projen' will generate unexpected outputs and the code will not build.